### PR TITLE
NEWS: add a belated NEWS file with a few entries for 0.9.1

### PR DESCRIPTION
**Pull request by trofi**
The main focus of this release is on medialib migration.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+commits. Notable changes:
+
+* Bundled waf was updated to 2.0.22 version. It now requires python3
+  support.
+
+* ffmpeg plugin was updated to support upstream API changes.
+
+* Medialib database switched from sqlite3 backend to s4 library backend.
+
+  For typical xmms2 setups that run in home directories backend
```
Comment (nesciens) [inline NEWS:15]:
```suggestion
  For typical xmms2 setups that run in home directories, backend
```

**Review by nesciens — APPROVED** [review]
Thanks for this, and for testing the migration!

Mostly non-blocking textual comments from me, and a question about sqlite2s4.

Please reject the textual tweaks if you feel they erase your tone of voice.

---

This is my first review via GitHub, so I might be holding this wrong. Tips for doing this more smoothly welcome!

File: NEWS [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  migration will happen automatically: once new version of `xmms2d`
+  starts it will convert ~/.config/xmms2/medialib.db sqlite database
+  to ~/.config/xmms2/medialib.s4 s4 database by running `sqlite2s4`
```
[the remark this answers is not in the document]
Comment (trofi) [reply NEWS]: Good idea! Added a standing out bullet point right after migration section as:

```
* New `sqlite2s4` tool was added. `xmms2d` expects it's presence in
  `PATH` to perform automatic medialib conversion to s4 format.
```

**Comment by trofi** [comment]
> Mostly non-blocking textual comments from me, and a question about sqlite2s4.

Pulled all you suggestions in, added a few newlines and squashed into a single commit. Please take another look.

Thank you!

File: NEWS [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  medialib are blocked. For an example library of 30000 entries (~78MB
+  medialib.db), it took 8 minutes to migrate to s4 on SSD. The
+  conversion uses fsync() calls extensively and might take longer on
+  HDD.
+
+  Once the conversion is done, the sqlite database will move to
+  ~/.config/xmms2/medialib.db.obsolete.
+
+* New `sqlite2s4` tool was added. `xmms2d` expects it's presence in
+  `PATH` to perform automatic medialib conversion to s4 format.
```
Comment (nesciens) [inline NEWS:39]:
```suggestion
* The new `sqlite2s4` tool was added. `xmms2d` expects its presence in
  `PATH` to perform automatic medialib conversion to the s4 format.
```

**Review by nesciens — APPROVED** [review]
A suggestion for the commit message (textual and collection vs medialib):

```
- Main focus of ths release is on collection migration.
+ The main focus of this release is on medialib migration.
```

Otherwise looks good to me.

**Comment by trofi** [comment]
> A suggestion for the commit message (textual and collection vs medialib):
> 
> ```
> - Main focus of ths release is on collection migration.
> + The main focus of this release is on medialib migration.
> ```

Good catch! Updated commit message and PR description.

Thank you!

**Review by nesciens — APPROVED** [review]
LGTM! 👍

**Pull request merged**